mirgecom/initializers.py

```diff
@@ -262,6 +263,95 @@ def __call__(self, x_vec, *, t=0, eos=IdealSingleGas()):
                               momentum=mom)
 
 
+class DoubleMachReflection:
+    r"""Implement the double shock reflection problem.
+
+        - Woodward and Collela
+
+    The inital condition is defined
+
+    .. math::
+
+        (\rho,u,v,P) =
+
+    This function only serves as an initial condition
+
+    .. automethod:: __init__
+    .. automethod:: __call__
+    """
+
+    def __init__(
+            self, dim=2, x0=1.0/6.0, us=4.0
+    ):
+        """Initialize initial condition options.
+
+        Parameters
+        ----------
+        dim: int
+           dimension of domain, must be 2
+        x0: float
+           location of shock
+        us: float
+           shock speed
```

Comment:
It might be nice to rename `x0` -> `shock_location`, `us` -> `shock_speed`

Reply:
https://github.com/illinois-ceesd/mirgecom/pull/213/commits/40364ac41ce999e79573e5980e6f671e8d20851a renames the variables as suggested.